src/framework/uicomponents/qml/MuseScore/UiComponents/NumberInputField.qml

```diff
@@ -51,6 +51,11 @@ FocusScope {
     function ensureActiveFocus() {
         if (!root.activeFocus) {
             root.forceActiveFocus()
+        } else if (!textField.activeFocus)
+        {
```

Comment:
Quoting the [QML coding guidelines](https://github.com/musescore/MuseScore/wiki/CodeGuidelines#qml):
> The opening brace goes on the same line as the start of the statement (with no exceptions)

So change line 54 to
```cpp
} else if (!textField.activeFocus) {
```
and remove line 55

Reply:
Ah, my bad!

Reply:
All good :slightly_smiling_face: